tests/factories/search_data.py

```diff
@@ -35,7 +35,8 @@
 
 
 class SearchDataProvider(BaseProvider):
-    def search_phrase(self) -> str:
+    @staticmethod
```

Comment:
What is the reason for this change? This will probably work (providers don't have tests in our set), but the convention here is to use a normal method instead of staticmethod - it's done this way in all other custom providers in the project.

Reply:
I saw that the 'self' object was not used in that method, but I'll stick to the convention.